### PR TITLE
fix allow list comments

### PR DESCRIPTION
**Pull request by ceyonur**
Fixes allow list comments

### CODE REVIEW

**Review by aaronbuchwald — APPROVED** [review]
LGTM

File: precompile/allow_list.go [hunk 1]
```diff
@@ -23,9 +23,12 @@ const (
 )
 
 var (
-	AllowListNoRole  AllowListRole = AllowListRole(common.BigToHash(big.NewInt(0))) // No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
-	AllowListEnabled AllowListRole = AllowListRole(common.BigToHash(big.NewInt(1))) // Deployers are allowed to create new contracts
-	AllowListAdmin   AllowListRole = AllowListRole(common.BigToHash(big.NewInt(2))) // Admin - allowed to modify both the admin and deployer list as well as deploy contracts
+	// No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
+	AllowListNoRole AllowListRole = AllowListRole(common.BigToHash(big.NewInt(0)))
```
Comment (darioush) [inline precompile/allow_list.go:27]: nit: could we use `comon.Big0`, `Big1` and `Big2`?
(are these equivalent to `common.Hash{0}, common.Hash{1}, and common.Hash{2}`)?

**Review by darioush — APPROVED** [review]

File: precompile/allow_list.go [hunk 2]
```diff
@@ -23,9 +22,12 @@ const (
...
 var (
-	AllowListNoRole  AllowListRole = AllowListRole(common.BigToHash(big.NewInt(0))) // No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
-	AllowListEnabled AllowListRole = AllowListRole(common.BigToHash(big.NewInt(1))) // Deployers are allowed to create new contracts
-	AllowListAdmin   AllowListRole = AllowListRole(common.BigToHash(big.NewInt(2))) // Admin - allowed to modify both the admin and deployer list as well as deploy contracts
+	// No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
+	AllowListNoRole AllowListRole = AllowListRole(common.BigToHash(common.Big0))
+	// Enabled - allowed to use state-changing precompile functions without modifying status of other admins or enableds
+	AllowListEnabled AllowListRole = AllowListRole(common.BigToHash(common.Big1))
+	// Admin - allowed to modify both the admin and enabled list, as well as to use state-changing precompile functions
+	AllowListAdmin AllowListRole = AllowListRole(common.BigToHash(common.Big2))
```
Comment (darioush) [inline precompile/allow_list.go:30]: is it necessary to write `AllowListAdmin AllowListRole = ...` or can the type be inferred eg `AllowListAdmin = ...`?

Comment (ceyonur) [reply precompile/allow_list.go:30]: yep good call

**Pull request merged**